index.d.ts

```diff
@@ -8,6 +8,7 @@ interface apiObject {
     | string
     | number
     | boolean
+    | undefined
     | apiObject
     | (string | number | boolean | apiObject)[];
```

Comment:
@boaz-codota shouldn't we support `undefined` here too?

Reply:
Why? What would anyone benefit from a list of undefined?

Reply:
One remote edge case I can think of is when the array is prepared from a bunch of other variables and one of them is `undefined`. At least, I haven't seen such a case but just struck me when I saw this update.

Reply:
Well @saikumarrs if this is a valid thing, then I suppose it could be addressed in a different PR. I would just love getting my PR merged and a update to a newer version.